source/plugins/ruby/KubernetesApiClient.rb

```diff
@@ -104,6 +108,7 @@ def getKubeResourceInfoV2(resource, api_group: nil)
               response = http.request(kubeApiRequest)
               responseCode = response.code
               @Log.info "KubernetesAPIClient::getKubeResourceInfoV2 : Got response of #{response.code} for #{uri.request_uri} @ #{Time.now.utc.iso8601}"
+              @@K8sApiResponseTelemetryTimeTracker = ApplicationInsightsUtility.sendAPIResponseTelemetry(response.code, resource, "K8sAPIStatus", @@K8sApiResponseCodeHash, @@K8sApiResponseTelemetryTimeTracker)
```

Comment:
Hey @ganga1980  that is because the cluster I was testing on had expired token so all the plugins were not running....Here is a working cluster

![image](https://github.com/microsoft/Docker-Provider/assets/18661789/a5d8634f-dad2-4f92-901d-05539359b47a)